server/src/main/java/org/elasticsearch/index/translog/Translog.java

```diff
@@ -1550,26 +1551,20 @@ static void verifyChecksum(BufferedChecksumStreamInput in) throws IOException {
      * use {@link #readOperations(StreamInput, String)} to read it back.
      */
     public static void writeOperations(StreamOutput outStream, List<Operation> toWrite) throws IOException {
-        final ReleasableBytesStreamOutput out = new ReleasableBytesStreamOutput(BigArrays.NON_RECYCLING_INSTANCE);
```

Comment:
This was the one place where we reset a releasable bytes output but it's obviously a bogus spot since it's based on non-recycling.

Reply:
I wonder how much we really gain from skipping around like this vs just writing two things to the stream. We only seem to use this code when serialising a batch of operations to send over the wire for recoveries anyway so it's already buffered.

Reply:
You're right ... that's kind of bogus. I cleaned it up to use two separarte writes.